### ISSUE TITLE
Mobile in-game chat complete disaster

### ISSUE THREAD

**Issue by tb2249** [SaitoTech/saito-lite-rust#2272]
overlapping windows. Defective style sheet mess. Complete disfunction. 
Tested on ios

Do I need to attach a photo? Any dev here can see for themselves anytime. 

Posted on Red Square and was told to fuck off, essentially, or post my comment here. Was a very disturbing way for someone on team Saito to respond to feedback. Actually I thought no need for feedback really, since it should be obvious ??
But apparently not. 

in summary:  normal desktop browser experience is ugly but passable.  Mobile is a joke; one that is somehow blended with netscape 1995 aesthetics.

**Comment by f8daniel** [comment]
Actually a photo would be helpful so we can tell (regardless of aesthetic choices) whether it is being rendered as designed or there is a bug.

"Tested on ios" -- ah, yes we have a ticket item for iPhones
